src/bin/nydus-image/main.rs

```diff
@@ -255,7 +255,7 @@ fn prepare_cmd_args(bti_string: String) -> ArgMatches<'static> {
                         .help("algorithm to digest inodes and data chunks:")
                         .takes_value(true)
                         .required(false)
-                        .default_value("blake3")
+                        .default_value("sha256")
```

Comment:
It seems sha256 is slower than blake3.

Reply:
We prefer compatibility instead of speed here:)